### ISSUE TITLE
Removing invalid parameter for timezone and datestyle

### ISSUE THREAD

**Issue by jouir** [zalando/patroni#649]
Hello,

When you configure client settings in postgresql.conf like `timezone` and `datestyle` in the postgresql.parameters' section of patroni, you end up with the following warnings at reload:
```
WARNING: Removing invalid parameter `timezone` from postgresql.parameters
WARNING: Removing invalid parameter `datestyle` from postgresql.parameters
```
The message is thrown because they are not present in the `pg_settings` table, because they are client settings but they are also valid postgresql.conf settings.

In practice, those settings are written in local files and PostgreSQL load them perfectly but the message could be misleading. Can we safely ignore the two warnings?

**Comment by CyberDem0n** [comment]
No, you can't ignore it. It actually means those settings are not applied...
The problem is that in pg_settings those parameters are called `TimeZone` and `DateStyle` accordingly.

The easiest fix for you would be to fix a config by changing the name.
Actually there is at least one more parameter, which name is written in CamelCase: https://www.postgresql.org/docs/current/static/runtime-config-client.html#GUC-DATESTYLE

It looks like we should really do case insensitive comparison.

**Comment by jouir** [comment]
It works using `TimeZone` and `DateStyle`. Thanks!
